src/lib/assetVersioning.js

```diff
@@ -0,0 +1,100 @@
+const fs = require('fs').promises;
+const path = require('path');
+const parseArguments = require('minimist');
+
+function main() {
+    let args = parseArguments(process.argv.slice(2));
+    let version = args['version'];
+    let assetTypes = [].concat(args['assetType']);
+    moveAssetFiles(version, assetTypes)
+        .catch(err => console.log(err));
+    updateVersionedDocsPaths(version, assetTypes)
+        .catch(err => console.log(err));
+}
```

Comment:
This doesn't seem right. Is this an outdated commit? You're using `commander` to process CLI parameters already and not `minimist`.